code/modules/mechs/mech_interaction.dm

```diff
@@ -1,4 +1,8 @@
 /mob/living/exosuit/MouseDrop_T(atom/dropping, mob/user)
+	if (user.buckled)
+		to_chat(user, SPAN_WARNING("You cannot enter a mech while buckled, unbuckle first."))
```

Comment:
?

Reply:
Okay, I manually went in there and deleted those changes. It says at the top that only 1 file should be changed now. If there's a different way to do it, let me know.